src/script/browser/CheckBrowser.ts

```diff
@@ -58,49 +58,60 @@ const supportsCookies = (): boolean => {
 const supportsIndexDB = (): Promise<boolean> =>
   new Promise<boolean>((resolve, _reject) => {
     if (!('indexedDB' in window)) {
-      resolve(false);
-    } else {
-      // some versions of FF don't allow access to IndexDB in the private mode
-      if (navigator.userAgent.toLowerCase().indexOf('firefox') !== -1) {
-        let dbOpenRequest: IDBOpenDBRequest;
+      return resolve(false);
```

Comment:
**question**: Did we change any logic in `supportsIndexDB` or it's just the refactoring?

Reply:
I did change the logic there, as it was returning `true` in one of the branches when it had to be `false`, and then I also refactored the code to be simpler